cli/cli_app.go

```diff
@@ -233,7 +233,7 @@ func downloadModules(terragruntOptions *options.TerragruntOptions) error {
 			return err
 		}
 		if shouldDownload {
-			return shell.RunTerraformCommand(terragruntOptions, "get", "-update")
+			return shell.RunTerraformCommandAndIgnoreOutput(terragruntOptions, "get", "-update")
```

Comment:
I don't think throwing the output away is a good approach. This output isn't useful during normal runs, but it's very useful for debugging issues.

The proper approach here is to use different levels of logging. We set the default debug level to INFO and output like this would be logged at the DEBUG level so that, normally, it doesn't show up. However, you could set the debug level to DEBUG or TRACE via config or a command-line flag to see more to help with debugging. 

Something like [go-logging](https://github.com/op/go-logging) would probably make this straightforward.